blazor/datagrid/column-menu.md

```diff
@@ -70,6 +70,87 @@ The default menu items are displayed in the following table,
 The following image represents DataGrid with column menu property enabled,
 ![Blazor DataGrid with Column Menu](images/blazor-datagrid-column-menu.gif)
 
+## Custom column menu
+
+You can customize the column menu by using the [ColumnMenuItems](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_ColumnMenuItems) property and the [ColumnMenuItemClicked](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_ColumnMenuItemClicked) event of the Grid.
```

Comment:
description is not enough its mean like just said the property names. Need to mention like using ColumnMenuItems property you can add your own customized item along with built in items or will add as new items in menu. Using ColumnMenuItemClicked event you can define the own custom logic or action while click on that customized item. 

The above sentence is for example i shared like that you need to document for all topics

Reply:
content changed as per your suggestion